src/slack/getSlackUserFromName.ts

```diff
@@ -0,0 +1,31 @@
+import {info} from '@actions/core'
+import {MemberWithProfile} from './types'
+
+/**
+ * Return the Slack user with the specified GitHub `name`.
+ *
+ * @throws {Error} if a user is not found, or if multiple users have the same `name`
+ */
+export function getSlackUserFromName(
+  users: MemberWithProfile[],
+  name: string | null
+): MemberWithProfile {
```

Comment:
a43deadcf18c8e8c9af8900c8e44d6f65994131f moves this shared function to a separate module.

Additionally, `SlackClient.getRealUsers()` now only returns members with profiles, simplifying caller logic.